README.md

```diff
@@ -0,0 +1,17 @@
+1. Создать класс «Сотрудник» с полями: ФИО, должность, email, телефон, зарплата, возраст;
```

Comment:
If you use sub-folders, your readme files won't conflict.

Reply:
I resolved all the conflicts with them